src/components/MediaPlayer/VideoJS/VideoJSUtils.js

```diff
@@ -0,0 +1,15 @@
+/** VideoJS custom components */
+import VideoJSProgress from './components/js/VideoJSProgress';
+import VideoJSCurrentTime from './components/js/VideoJSCurrentTime';
+import VideoJSFileDownload from './components/js/VideoJSFileDownload';
+import VideoJSNextButton from './components/js/VideoJSNextButton';
+import VideoJSPreviousButton from './components/js/VideoJSPreviousButton';
+// import vjsYo from './vjsYo';
+
+/** VideoJS language files */
+import de from "video.js/dist/lang/de.json";
+import en from "video.js/dist/lang/en.json";
+import fr from "video.js/dist/lang/fr.json";
+import it from "video.js/dist/lang/it.json";
+
+export const LANG_MAP = { "de": de, "fr": fr, "it": it, "en": en };
```

Comment:
If the language used in the interface is based upon a prop then it would be nice to dynamically load the language json file if that's possible.  If not, I wonder if it might be helpful to load all of the languages here if that doesn't have too large of a performance impact.  If they are all loaded then all of the options are available to the implementor without having to modify ramp.